wordpress-mobile/WordPress-Android
# Changes domain placeholder to yourgroovydomain.com

Reference p1701163435325639-slack-C05NS0YV7HS

## Description
Changes the domain placeholder of the "Free domain with annual plan" card from `domain.blog` to `yourgroovydomain.com`.

|Before|After|
|---|---|
|![Screenshot_20231129_122138](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/c941443b-0249-4959-8a44-fb847a903445)|![Screenshot_20231129_122407](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/87654142-c77d-4b09-ab30-55266ea0c9cb)|

|Before Dark Theme|After Dark Theme|
|---|---|
|![Screenshot_20231129_122156](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/960026fb-679a-4837-a391-95ed547d6b81)|![Screenshot_20231129_122350](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/77466eae-c451-4a73-a84f-48200ca2f21a)|

|Landscape|Large fonts|
|---|---|
|![Screenshot_20231129_122430](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/bef34214-63ff-4c11-9360-1fed63f4cf4a)|![Screenshot_20231129_122531](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/f9f784a7-da70-49bd-ac90-933f02db4790)|

-----

## To Test:
1. Open the Jetpack app
2. Scroll down the *My site* screen
3. **Verify** that the placeholder domain is `yourgroovydomain.com`

-----

## Regression Notes

1. Potential unintended areas of impact

N/A

4. What I did to test those areas of impact (or what existing automated tests I relied on)

Manual testing

5. What automated tests I added (or what prevented me from doing so)

This PR involves only a resource change

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
